deploy-site.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+echo -e "\033[0;32mDeploying website...\033[0m"
+
+cd site && npx hexo generate
+
+# delete old deploy branch
+git branch -D deploy
```

Comment:
I'm wondering if it might be safer to "scope" the branch names: `embark-site-deploy` and `embark-site-deploy-dist`.  I don't think `deploy` and `deploy-dist` will conflict with anyone's local branches, but maybe better to play it safe.

Reply:
Good point!